File: Data Structures & Algorithms/length-of-last-word/submission-1.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int length = 0;
        for(int x = s.length() - 1; x >= 0; x--){
            if(s[x] == ' ' && length == 0) continue;
            if(s[x] == ' ' && length > 0) return length;
            length++;
        }

        return length;
    }
};

File: Data Structures & Algorithms/baseball-game/submission-1.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        stack = []
        for operation in operations:
            if operation == "+":
                if len(stack) >= 2:
                    newVal = int(stack[-1]) + int(stack[-2])
                    stack.append(newVal)
            elif operation == "D":
                if stack:
                    stack.append(int(stack[-1])*2)
            elif operation == "C":
                if stack:
                    stack.pop()
            else:
                stack.append(int(operation))

        return sum(stack)